### PR TITLE
feat(app): update style for ChangePipette confirm Attach/Detach screens

### PR DESCRIPTION
**Pull request by sakibh**
# Overview

This PR updates the styling of the ChangePipette components confirm Attach and Detach screens. closes [RLIQ-46](https://opentrons.atlassian.net/browse/RLIQ-46?atlOrigin=eyJpIjoiZjYxYTExZWY4NzA3NGI0N2IwZjMyNmM2MWEzM2ExOWQiLCJwIjoiaiJ9)

- Update `ConfirmPipette` styling and deprecate old component
- Update tests

# Review requests

- Smoke test on a robot to ensure the screens below work
- Detach pipette successfully
- Detach pipette without physically detaching to see the pipette still detected screen
- Attach a pipette but physically attach the an incorrect pipette to see the incorrect pipette was attached screen
- Attempt to attach a pipette without physically attaching it, should show no pipette detected
- Attach a pipette successfully (POC data should exist already)
- While POC data is deleted, attach a pipette successfully which renders the `Calibrate pipette offset` button.

# Risk assessment
low, behind FF

### CODE REVIEW

File: app/src/organisms/ChangePipette/ConfirmPipette.tsx [hunk 1]
```diff
@@ -13,17 +15,13 @@ import type {
 } from '@opentrons/shared-data'
 import type { Mount } from '../../redux/pipettes/types'
 import type { PipetteOffsetCalibration } from '../../redux/calibration/types'
+import { SimpleWizardModal } from '../../molecules/SimpleWizardModal'
+import { PrimaryButton, SecondaryButton } from '../../atoms/buttons'
```
Comment (jerader) [inline app/src/organisms/ChangePipette/ConfirmPipette.tsx]: these imports can go above the type imports

File: app/src/organisms/ChangePipette/ConfirmPipette.tsx [hunk 2]
```diff
@@ -32,191 +30,133 @@ export interface Props {
...
-      onClick={props.startPipetteOffsetCalibration}
+    <SimpleWizardModal
+      iconColor={success ? COLORS.successEnabled : COLORS.errorEnabled}
+      header={header}
+      subHeader={subHeader}
+      isSuccess={success}
+      onExit={exit}
+      title={title}
+      currentStep={5}
+      totalSteps={8}
```
Comment (jerader) [inline app/src/organisms/ChangePipette/ConfirmPipette.tsx]: I think we want these 2 plugs in as props so we can wire up all the `currentStep` and `totalSteps` in `ConfirmPipette`

File: app/src/organisms/ChangePipette/Instructions.tsx [hunk 3]
```diff
@@ -17,6 +18,8 @@ import { PrimaryButton } from '../../atoms/buttons'
 import { CheckPipettesButton } from './CheckPipettesButton'
 import { InstructionStep } from './InstructionStep'
 import { PipetteSelection } from './PipetteSelection'
+import { ModalShell } from '../../molecules/Modal'
```
Comment (jerader) [inline app/src/organisms/ChangePipette/Instructions.tsx]: this import can be higher

File: app/src/organisms/ChangePipette/Instructions.tsx [hunk 4]
```diff
@@ -168,11 +171,32 @@ export function Instructions(props: Props): JSX.Element {
...
+        <ModalShell height="28.12rem" width="47rem">
+          <LevelPipette
+            mount={mount}
+            wantedPipette={wantedPipette}
+            pipetteModelName={actualPipette ? actualPipette.name : ''}
+            exit={exit}
+            confirm={confirm}
+            back={back}
+            currentStep={6}
+            totalSteps={8}
```
Comment (jerader) [inline app/src/organisms/ChangePipette/Instructions.tsx]: these 2 can be plugged into the props

File: app/src/organisms/ChangePipette/index.tsx [hunk 5]
```diff
@@ -331,8 +323,30 @@ export function ChangePipette(props: Props): JSX.Element | null {
...
         <ConfirmPipette
+          {...{
+            ...basePropsWithPipettes,
+            title:
+              actualPipette?.displayName != null
+                ? t('detach_pipette', {
+                    pipette: actualPipette.displayName,
+                    mount: mount[0].toUpperCase() + mount.slice(1),
+                  })
+                : t('attach_pipette'),
```
Comment (jerader) [inline app/src/organisms/ChangePipette/index.tsx]: This logic is a bit wrong, we also want to check for if we have a `wantedPipette` too.

**Review by jerader — COMMENTED** [review]
Looking pretty good so far! I left some comments. I also tested this on my robot and the main thing I noticed was the wizard headers were incorrect most of the time, otherwise functionality was as expected.

Should say `attach` instead of `detach`
Screen Shot 2022-08-25 at 9 11 56 AM

Should say `detach pipette`
Screen Shot 2022-08-25 at 9 13 04 AM

Should say `Detach {{pipetteName}} from {{mount}} Mount`
Screen Shot 2022-08-25 at 9 13 38 AM

**Comment by jerader** [comment]
Also are you missing a test for `ConfirmPipette`?

**Comment by jerader** [comment]
also, testing out the `PipetteLeveling` and got this white screen when I said "Confirm attachment" but didn't actually attach the pipette:

Screen Shot 2022-08-25 at 9 29 44 AM

File: app/src/organisms/ChangePipette/ConfirmPipette.tsx [hunk 6]
```diff
@@ -32,191 +30,133 @@ export interface Props {
...
+    <>
+      {success && actualPipette && !actualPipetteOffset && (
+        <SecondaryButton
+          marginRight={SPACING.spacing3}
+          onClick={startPipetteOffsetCalibration}
+        >
+          {t('calibrate_pipette_offset')}
+        </SecondaryButton>
+      )}
+      <PrimaryButton onClick={exit}>{t('shared:exit')}</PrimaryButton>
```
Comment (jerader) [inline app/src/organisms/ChangePipette/ConfirmPipette.tsx]: this `exit` text needs to be capitalized

File: app/src/organisms/ChangePipette/index.tsx [hunk 7]
```diff
@@ -37,10 +37,11 @@ import { ClearDeckModal } from './ClearDeckModal/index'
 import { ExitAlertModal } from './ExitAlertModal'
 import { DeprecatedInstructions } from './DeprecatedInstructions'
 import { ConfirmPipette } from './ConfirmPipette'
+import { DeprecatedConfirmPipette } from './DeprecatedConfirmPipette'
 import { RequestInProgressModal } from './RequestInProgressModal'
-import { LevelPipette } from './LevelPipette'
 import { DeprecatedLevelPipette } from './DeprecatedLevelPipette'
 import { ClearDeckAlertModal } from './ClearDeckModal/ClearDeckAlertModal'
+import { Mount, SPACING } from '@opentrons/components'
```
Comment (jerader) [inline app/src/organisms/ChangePipette/index.tsx]: Add the `type` lower with the other `types`

```suggestion
import { SPACING } from '@opentrons/components'
import type { Mount } from '@opentrons/components'
```

File: app/src/organisms/ChangePipette/ConfirmPipette.tsx [hunk 8]
```diff
@@ -32,191 +29,155 @@ export interface Props {
   displayName: string
   displayCategory: PipetteDisplayCategory | null
   tryAgain: () => unknown
-  back: () => unknown
   exit: () => unknown
```
Comment (jerader) [inline app/src/organisms/ChangePipette/ConfirmPipette.tsx]: these were already here but lets change them to `() => void`

File: app/src/organisms/ChangePipette/index.tsx (hunk 7 above)
Comment (jerader) [reply app/src/organisms/ChangePipette/index.tsx]: Also wait, is that the right import for `Mount`, Shouldn't it be imported from `redux/pipettes/types`?

File: app/src/organisms/ChangePipette/Instructions.tsx [hunk 9]
```diff
@@ -168,11 +171,34 @@ export function Instructions(props: Props): JSX.Element {
         {stepPage === 0 ? (
           continueButton
         ) : (
-          <CheckPipettesButton robotName={robotName} onDone={confirm}>
+          <CheckPipettesButton
```
Comment (jerader) [inline app/src/organisms/ChangePipette/Instructions.tsx:174]: We should add a test case for these additions

File: app/src/organisms/ChangePipette/Instructions.tsx [hunk 10]
```diff
@@ -168,11 +171,34 @@ export function Instructions(props: Props): JSX.Element {
...
+                : confirm
+            }
+          >
             {actualPipette ? t('confirm_detachment') : t('confirm_attachment')}
           </CheckPipettesButton>
         )}
       </Flex>
+      {stepPage === 2 && (
+        <ModalShell height="28.12rem" width="47rem">
+          <LevelPipette
```
Comment (jerader) [inline app/src/organisms/ChangePipette/Instructions.tsx:190]: also add a test case for this too

File: app/src/organisms/ChangePipette/index.tsx (hunk 7 above)
Comment (sakibh) [reply app/src/organisms/ChangePipette/index.tsx]: They both seem to be same exact thing. I will switch the import to the redux directory though.

File: app/src/organisms/ChangePipette/Instructions.tsx (hunk 10 above)
Comment (sakibh) [reply app/src/organisms/ChangePipette/Instructions.tsx:190]: Will address these test cases in a follow up

**Review by jerader — APPROVED** [review]
lgtm!! ✅

**Pull request merged**